src/components/FormElements/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,12 @@
+.Checkbox {
+  color: var(--color-gray-700);
+  display: inline-flex;
```

Comment:
btw, should we start using styled-components from now onwards?

Reply:
after this PR, yeah 😆